src/settings/index.html

```diff
@@ -26,6 +31,14 @@
               <input type="text" id="airtable_key" placeholder="keyXXXXXXXXX">
             </td>
           </tr>
+          <tr>
+            <td colspan="2">
+              <label for="show_pm_factor_field">
+                <input type="checkbox" id="show_pm_factor_field" name="show_pm_factor_field" value="1">
```

Comment:
`value="1"` doesn't define the default value because it's not defined (or missing) in the `browser.storage` and we reset the `checked` (settings.js, line 18) based on it. Maybe we can add a test in the `settings.js` and set a default checked states for this option if it's undefined 😉

Reply:
we can remove the `value=1` attribute as we only test the `checked` attribute.
During my test, I didn't understand why I always got the value `on` when checking `my_radio.value` so I changed my mind and based the test on `checked`.